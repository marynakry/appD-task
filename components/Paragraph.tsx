import styled from 'styled-components'

export const Paragraph = styled.p`
  font: 13px Helvetica, Arial;
  margin: 10px 0;
`
