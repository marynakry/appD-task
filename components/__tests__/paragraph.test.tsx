import { Paragraph } from '../Paragraph'
import React from 'react'
import { render } from '@testing-library/react'
import { describe, expect, it } from '@jest/globals'

describe('Paragraph', () => {
  it('render paragraph component', () => {
    const { getByText } = render(<Paragraph>test</Paragraph>)
    expect(getByText('test')).toBeTruthy()
    expect(getByText('test')).toMatchSnapshot()
  })
})
