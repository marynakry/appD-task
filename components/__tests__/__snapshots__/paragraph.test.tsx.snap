// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph render paragraph component 1`] = `
<p
  class="Paragraph-arz900-0 llfRtB"
>
  test
</p>
`;
